django_loci/apps.py

```diff
@@ -15,11 +23,21 @@ def __setmodels__(self):
         from .models import Location
         self.location_model = Location
 
+    def test_geocode(self):
+        location = GeoCodeView.geocode('Red Square')
```

Comment:
don't use the view. Move the logic which does geocoding out of the view in a function or a few functions or a class or whatever and call the same code from the view, as explained in the issue description